cmd/brew-portable-package.rb

```diff
@@ -20,7 +20,7 @@
   safe_system "brew", "test", name
   puts "Library linkage:"
   if OS.linux?
-    puts Utils.open_read "ldd", "#{f.bin}/#{name.gsub(/^portable-/, "")}"
+    puts Utils.popen_read "ldd", "#{f.bin}/#{name.gsub(/^portable-/, "")}"
```

Comment:
Ah, right. Good point. I have a conference in week. After I'm back from that I'll send some patches upstream.